modules/motions/Straight.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file   Straight.cpp
+ * @brief  直進動作の親クラス
+ * @author HaruArima08
+ */
+
+#include "Straight.h"
+using namespace std;
+
+Straight::Straight(Robot& robot, double _targetSpeed) : Motion(robot), targetSpeed(_targetSpeed) {}
+void Straight::run()
```

Comment:
```suggestion
Straight::Straight(Robot& robot, double _targetSpeed) : Motion(robot), targetSpeed(_targetSpeed) {}

void Straight::run()
```
関数の前後は一行あけないと clang-format 通らないかも。